debian/patches/device-fixes-nissan-connect.patch

```diff
@@ -0,0 +1,13 @@
+--- a/plugins/autopair.c
++++ b/plugins/autopair.c
+@@ -78,6 +78,10 @@
+ 	if (name != NULL && strstr(name, "iCade") != NULL)
+ 		return 0;
+ 
++	/* Nissan Connect carkits use PIN 1234 but refuse a retry */
++	if (name != NULL && strstr(name, "NISSAN CONNECT") != NULL)
```

Comment:
FYI: https://github.com/bluez/bluez/commit/823821dca098018e5fee9c8556c5d0896c65b972

but it matches code style of our current state, so should be fine but when submitting upstream this should match upstream and `name != NULL` should be removed.